atest/testdata/parsing/test_case_settings.robot

```diff
@@ -214,14 +210,19 @@ Multiple settings
     [Teardown]    Log    Test case teardown
 
 Invalid setting
+    [Documentation]    FAIL Non-existing setting 'Invalid'.
     [Invalid]    This is invalid
     No Operation
 
 Setting not valid with tests
+    [Documentation]    FAIL Setting 'Metadata' is not allowed with tests or tasks.
     [Metadata]    Not valid.
     [Arguments]    Not valid.
     No Operation
 
 Small typo should provide recommendation
+    [Documentation]    FAIL
+    ...    Non-existing setting 'Doc U ment a tion'. Did you mean:
+    ...    ${SPACE*4}Documentation
     [Doc U ment a tion]    This actually worked before RF 3.2.
     No Operation
```

Comment:
`No Operation` could be changed to something like `Fail    Should not be run` in cases like this.